app/code/Magento/Checkout/view/frontend/web/js/model/shipping-save-processor/payload-extender.js

```diff
@@ -0,0 +1,9 @@
+define([], function () {
```

Comment:
Please add copyright block to this file (would be nice if you will join all changes in single commit).

Reply:
I've made the changes as requested.  Also removed the [], since other Magento core JavaScript files seem to, and it's fairly useless.  Amended it to the previous commit and force-pushed so it should still all be one commit.